pos_barcode_tare/static/src/xml/pos_barcode_tare.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="TareScreenWidget">
+        <div class='tare-screen screen'>
+            <div class='screen-content'>
+                <div class='top-content'>
+                     <span class='button back'>
+                        <i class='fa fa-angle-double-left'></i>
+                        Back
+                    </span>
+                    <h1>Print a tare label</h1>
+                </div>
+                <div class="centered-content">
+                    <div class="pos-tare-label-container"></div>
+                    <div class="pos-directions-for-user">
+                        Set the pot on the scale and check the weight above
+                    </div>
+                    <div class='print-label'>
+                        Print
+                        <i class='fa fa-angle-double-right'></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="PosTareLabel">
+        <div class="pos-center-align">
+            <div class="pos-tare-paper">
+                <div class="pos-tare-label">
+                    <img t-att-src="'/report/barcode/EAN13/' + widget.get_barcode_data()" />
+                    <span class="caption">tare = <t t-esc="widget.get_weight()" />kg</span>
```

Comment:
maybe set the barcode also. (I saw other sytem where the barcode was written)

something like : 
                     <span class="caption"><t t-esc="widget.get_barcode_data()" /></span>

Reply:
Not sure about this. To my understanding, the rationale for having the barcode data written is to enable the operator to manually input the barcode if the scanner fails to decode it. Do you think this applies to POS? If so I'm happy to print the barcode.

Reply:
Hello @legalsylvain, 

Thank you for the thoughtful review!

I'll (try to) do the same for #436. I agree with you #447 and #436 are complementary and should merge at some point! 

Thank you,
François

> Hi @Fkawala.
> 
> thanks a lot for sharing this module.
> 
> some comments inline. no important blockings points.
> 
> FYI, I wrote a module for V12, named `pos_tare`. Complementary features.
> maybe we could imagine to merge that feature in a second step.
> 
> could you take a look, and make a review ?
> Thanks !
> #436